Gemfile

```diff
@@ -26,12 +26,16 @@ group :development do
   # The latest version of rubocop is only compatible with Ruby 2.7+
   gem "rubocop", "1.57.2" if RUBY_VERSION >= "2.7"
 
-  gem "sorbet"
-  gem "tapioca"
+  gem "sorbet", require: false
+  gem "tapioca", require: false
```

Comment:
Any reason we want this extra changes instead of using what is in #1549 for the beta branch?